src/server.js

```diff
@@ -52,13 +54,28 @@ export default class TapestryServer {
     })
   }
 
+  routeApi () {
```

Comment:
It's a basic, and slow implementation, I think we'll write a `proxyHandler` method that does more involved logic, like rewriting headers, and maybe a bit of Tapestry-side caching.